shell/platform/tizen/tizen_renderer_ecore_wl2.cc

```diff
@@ -296,7 +423,17 @@ bool TizenRendererEcoreWl2::SetupEcoreWlWindow(int32_t width, int32_t height) {
 
   ecore_wl2_window_ =
       ecore_wl2_window_new(ecore_wl2_display_, nullptr, x, y, width, height);
-  ecore_wl2_window_type_set(ecore_wl2_window_, ECORE_WL2_WINDOW_TYPE_TOPLEVEL);
+  ecore_wl2_window_type_set(ecore_wl2_window_,
+                            top_ ? ECORE_WL2_WINDOW_TYPE_NOTIFICATION
+                                 : ECORE_WL2_WINDOW_TYPE_TOPLEVEL);
+  if (top_) {
+    while (!tizen_policy_) {
+      wl_display_dispatch_queue(wl_display_, wl_event_queue_);
+    }
+    tizen_policy_set_notification_level(
```

Comment:
Is this the part we need the most? It's not enough to set the window type like above.

Reply:
The notification level is only effective when the window type is notification.